### DISCUSSION TITLE
I have a project ready to publish, but I need to fix an issue. How do you change a "tag" collection to a "book"?

### DISCUSSION THREAD

**Discussion by wcp200** [knowledgefutures/pubpub#2842] [Support]
Hi everyone. I have a project ready to publish, but I need to fix an issue. How do you change a "tag" collection to a "book"?

**Comment by gabestein** [comment]
Unfortunately, this isn't possible in the UI, as the types have very different metadata structures behind the scenes. If you write in to hello@pubpub.org with a link to the collection, we can change it for you in the database, but you'll lose most tag settings as a result and need to re-do them.
